### PR TITLE
19652 Fix mailing/physical address combo problems

### PR DESCRIPTION
**Pull request by NateJohnson-BAH**
## Description
The comparison tool is displaying a combination of the physical address and mailing address on the search results and profile pages. (now, we've also added the 'school locations' section of the profile page to this PR)
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/19652

## Testing done
Confirmed locally in UI, via React Devtools to verify state and props were using correct info across various institutions

## Screenshots
Search Results showing in country, and out of country cards with appropriate info:
Screen Shot 2019-09-27 at 12 52 13 PM

Example institution profile page showing non-US country, as expected
Screen Shot 2019-09-27 at 12 52 28 PM

School locations table for non-US institution:
Screen Shot 2019-09-27 at 12 52 43 PM

School locations table for US institution with zip code:
Screen Shot 2019-09-27 at 12 52 58 PM

## Acceptance criteria
- [x] Desired behavior:
1. If the institution is in the US or US Territory
a. The physical address city & state are displayed on the search results card.
  i. if only city, or only state is provided, show only relevant formatted information
  ii. if no physical address information is available, show a blank string
b. The physical address city & state are displayed on the profile page.
  i. if only city, or only state is provided, show only relevant formatted information
  ii. if no physical address information is available, the icon with location info is not displayed
c. The physical address city & state & zipcode are displayed on the profile page 'school locations' table.
  i. if only city, or only state is provided, show only relevant formatted information
  ii. if no physical address information is available, show a blank string
 iii. show physical zip code, if available, but after city, state (if available)
2. If the institution is foreign
a. The physical address city & country are displayed on the search results card.
  i. if only city, or only country is provided, show only relevant formatted information
  ii. if no physical address information is available, show a blank string
b. The physical address city & country are displayed on the profile page.
  i. if only city, or only country is provided, show only relevant formatted information
  ii. if no physical address information is available, the icon with location info is not displayed
c. The physical address city & country are displayed on the profile page 'school locations' table.
  i. if only city, or only country is provided, show only relevant formatted information
  ii. if no physical address information is available, show a blank string
 
## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/gi/components/profile/HeadingSummary.jsx [hunk 1]
```diff
@@ -81,9 +81,18 @@ class HeadingSummary extends React.Component {
           </div>
           <div>
             <div className="usa-width-one-half medium-6 small-12 column">
-              <IconWithInfo icon="map-marker" present={it.city && it.country}>
-                {it.city}, {it.state || it.country}
-              </IconWithInfo>
+              {it.country === 'USA' ? (
```
Comment (zurbergram) [inline src/applications/gi/components/profile/HeadingSummary.jsx]: This variable name is not great

**Review by zurbergram — APPROVED** [review]
LGTM

File: src/applications/gi/components/profile/HeadingSummary.jsx (hunk 1 above)
Comment (NateJohnson-BAH) [reply src/applications/gi/components/profile/HeadingSummary.jsx]: Agreed. If I made more changes to the file, I would have fixed it. Didn't want to cause more potential issues. :)

**Comment by shawkey-daniel** [comment]
The first example doesn't look right. Is there a Panama, FL?

**Comment by NateJohnson-BAH** [comment]
@shawkey-daniel Yes, that's what is in the data:

Screen Shot 2019-09-23 at 11 39 10 AM

**Comment by shawkey-daniel** [comment]
> @shawkey-daniel Yes, that's what is in the data:
> 
> Screen Shot 2019-09-23 at 11 39 10 AM

I think that's bad data, but that's not part of this defect. Thanks.

**Comment by NateJohnson-BAH** [comment]
In an attempt to simplify and force correct data usage, we have defaulted to 'physicalState' and physicalCity' for both search and profile pages on a given institution. Also, I added conditional logic to only show data that is available. For example, if we only have a city, that is all that is displayed. If we have a city, state then we display such. If the country is not USA, we display city and country. If the data seems incorrect, it is most likely bad data in the source weams file. 

Updated search results with more varied data to display changes:
Screen Shot 2019-09-23 at 12 22 01 PM

http://localhost:3001/gi-bill-comparison-tool/profile/11816110 is the example link given in the defect. This is what it looks like now. 'PANAMA' is listed as the physicalCity in the weams data, so is correct based on current logic. 
Screen Shot 2019-09-23 at 12 26 52 PM

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]
Can we get normalizedAttributes into a single location instead of repeating the code in two files?

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by shawkey-daniel — APPROVED** [review]

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]
Can we combine the two locationInfo?

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: .gitignore [hunk 2]
```diff
@@ -32,3 +32,4 @@ temp
 .swo
 all-links.csv
 .tern-port
+*.iml
```
Comment (rianfowler) [inline .gitignore]: must be our first intelliJ user 👍

**Review by rianfowler — APPROVED** [review]

**Pull request merged**